src/libs/actions/PaymentMethods.ts

```diff
@@ -109,74 +122,80 @@ function getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMet
     return onyxData;
 }
 
+type MakeDefaultPaymentMethodParams = {
+    bankAccountID: number;
+    fundID: number;
+};
+
 /**
  * Sets the default bank account or debit card for an Expensify Wallet
  *
- * @param {Number} bankAccountID
- * @param {Number} fundID
- * @param {Object} previousPaymentMethod
- * @param {Object} currentPaymentMethod
- *
  */
-function makeDefaultPaymentMethod(bankAccountID, fundID, previousPaymentMethod, currentPaymentMethod) {
-    API.write(
-        'MakeDefaultPaymentMethod',
-        {
-            bankAccountID,
-            fundID,
-        },
-        {
-            optimisticData: getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMethod, currentPaymentMethod, true, ONYXKEYS.FUND_LIST),
-            failureData: getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMethod, currentPaymentMethod, false, ONYXKEYS.FUND_LIST),
-        },
-    );
+function makeDefaultPaymentMethod(bankAccountID: number, fundID: number, previousPaymentMethod: PaymentMethod, currentPaymentMethod: PaymentMethod) {
+    const parameters: MakeDefaultPaymentMethodParams = {
+        bankAccountID,
+        fundID,
+    };
+
+    API.write('MakeDefaultPaymentMethod', parameters, {
+        optimisticData: getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMethod, currentPaymentMethod, true),
+        failureData: getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMethod, currentPaymentMethod, false),
+    });
 }
 
+type AddPaymentCardParams = {
```

Comment:
You can move params type to function body (examples in Fabio's [PR](https://github.com/Expensify/App/pull/28269))